[Package]: hello

### Package description

Prints Hello World and more

### Home page URL

https://gnu.org/software/hello

### Source code URL

https://mirrors.kernel.org/gnu/hello/hello-${TERMUX_PKG_VERSION}.tar.gz

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

I try to build, but fail:

```
mv -f lib/configmake.h-t lib/configmake.h
rm -f lib/arg-nonnull.h-t lib/arg-nonnull.h && \
sed -n -e '/GL_ARG_NONNULL/,$p' \
  < /home/wzy/.termux-build/hello/src/build-aux/snippet/arg-nonnull.h \
  > lib/arg-nonnull.h-t && \
mv lib/arg-nonnull.h-t lib/arg-nonnull.h
rm -f lib/c++defs.h-t lib/c++defs.h && \
sed -n -e '/_GL_CXXDEFS/,$p' \
  < /home/wzy/.termux-build/hello/src/build-aux/snippet/c++defs.h \
  > lib/c++defs.h-t && \
mv lib/c++defs.h-t lib/c++defs.h
rm -f lib/warn-on-use.h-t lib/warn-on-use.h && \
sed -n -e '/^.ifndef/,$p' \
  < /home/wzy/.termux-build/hello/src/build-aux/snippet/warn-on-use.h \
  > lib/warn-on-use.h-t && \
mv lib/warn-on-use.h-t lib/warn-on-use.h
/bin/sh: line 31: lib/configmake.h-t: No such file or directory
/bin/sh: line 2: lib/arg-nonnull.h-t: No such file or directory
make: *** [Makefile:2653: lib/configmake.h] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:2792: lib/arg-nonnull.h] Error 1
/bin/sh: line 2: lib/warn-on-use.h-t: No such file or directory
/bin/sh: line 2: lib/c++defs.h-t: No such file or directory
make: *** [Makefile:2809: lib/warn-on-use.h] Error 1
make: *** [Makefile:2800: lib/c++defs.h] Error 1
```

--- a/packages/hello/build.sh
+++ b/packages/hello/build.sh
@@ -1,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/hello/
+TERMUX_PKG_DESCRIPTION="Prints a friendly greeting"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.12
+TERMUX_PKG_SRCURL=https://ftp.gnu.org/gnu/hello/hello-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=cf04af86dc085268c5f4470fbae49b18afbc221b78096aab842d934a76bad0ab
+TERMUX_PKG_DEPENDS="libiconv"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	LDFLAGS+=" -liconv"
+}

